composer.json

```diff
@@ -12,6 +12,7 @@
         }
     ],
     "autoload": {
+        "files": ["src/functions_include.php"],
```

Comment:
> It's a bit of an odd thing to have happen, but I'm not sure what would happen if a project had 2 dependencies that both depended on this lib. Composer uses include rather than include_once.

Nope, was just interested. :+1: Thanks.